__tests__/integration/configuration/docker-compose.yml

```diff
@@ -67,6 +70,172 @@ services:
     networks:
       - hathor-privnet
 
+
+  # Wallet Service section
+  # All the following services are related to the Wallet Service
+  # Redis on 6379, MySQL on 3306
+  # Wallet Service Daemon on 8081 and Daemon Websocket on 8082
+  # Wallet Service API Lambda on 3000 and Wallet Service Websocket Lambda on 3001
+
+  mysql:
+    image: centos/mysql-80-centos7 # Necessary to use specific authentication protocol
+    networks:
+      - hathor-privnet
+    environment:
+      MYSQL_ROOT_PASSWORD: hathor
+      MYSQL_DATABASE: wallet_service
+      MYSQL_USER: wallet_service_user
+      MYSQL_PASSWORD: password
+      MYSQL_DEFAULT_AUTHENTICATION_PLUGIN: mysql_native_password
+    ports:
+      - "3380:3306"
+    healthcheck:
+        test: ["CMD", "mysql", "-h", "mysql", "-u", "wallet_service_user", "-ppassword", "-e", "SELECT 1", "wallet_service"]
+
+  redis:
+    image: redis:6.2
+    networks:
+      - hathor-privnet
+    ports:
+      - "6379:6379"
+    healthcheck:
+      test: [ "CMD", "redis-cli", "ping" ]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+
+  ws-migrator:
+    build:
+      context: '../../../../wallet-service/'
+      dockerfile: 'db/Dockerfile'
```

Comment:
Those build instructions will be replaced with actual DockerHub image addresses as the PRs develop in the Wallet Service repository.

Reply:
This was already implemented. The only thing pending is now to publish the images for the CI to use.